include/color_spinor.h

```diff
@@ -981,6 +1003,28 @@ namespace quda {
     return dot;
   }
 
+  /**
+     Compute the cross product of two color vectors at spin sa and sb
+     cProd = \sum_{j,k} \epsilon_{i,j,k} a(s1,j) b(s2,k)     
+     @param a j ColorSpinor
+     @param b k ColorSpinor
+     @param sa j spin index
+     @param sb k spin index
+     @return The cross product
+  */
```

Comment:
This function takes templated Nc but appears to be hard coded for Nc=3.

Reply:
Good point! I'll remove the Nc=3 template... not in a hurry to abstract an Nc Levi-Civita.